lib/active_job/queue_adapters/sqs_adapter.rb

```diff
@@ -66,7 +66,8 @@ def _enqueue(job, body = nil, send_message_opts = {})
         params = Params.new(job, body)
         send_message_opts = send_message_opts.merge(params.entry)
         send_message_opts[:queue_url] = params.queue_url
-        Aws::ActiveJob::SQS.config.client.send_message(send_message_opts)
+        response = Aws::ActiveJob::SQS.config.client.send_message(send_message_opts)
+        job.provider_job_id = response.message_id
```

Comment:
Does the return value of this method matter? From what I can see, it should return the job/response or false.